<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1346</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>521</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>410</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-6</number>
    </property>
    <property name="text">
     <string>График населения областей Российской Федерации : </string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите области которые вы хотите сравнить: </string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>761</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1346</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
